btcjson/chainsvrresults.go

```diff
@@ -379,6 +382,48 @@ type GetRawMempoolVerboseResult struct {
 	Depends          []string `json:"depends"`
 }
 
+// ZmqNotificationResult models the data returned from the zmqnotifications command.
+type ZmqNotificationResult struct {
+	Type          string
+	Address       *url.URL
+	HighWaterMark int
```

Comment:
What do you think about adding some comments for the struct fields? I think it's particularly useful when identifiers do not convey enough meaning.

```suggestion
	Type          string    // Type of notification
	Address       *url.URL  // Address of the publisher
	HighWaterMark int       // Outbound message high water mark
```